### ISSUE TITLE
About train/valid splitt

### ISSUE THREAD

**Issue by akbacak** [metal3d/keras-video-generators#6]
I test your code till the line "valid = train.get_validation_generator()"
I have “videos” folder, it has two subfolders: a and b, each contains 30 mp4 files. Code gives me : 
class a, validation count: 0
class b, validation count: 0
get 2 classes for 2 files for train
get 2 classes for 0 files for validation

**Comment by metal3d** [comment]
OK, didn't see the issue.
Can you please tell me:
- if there is subfolders inside your class folders
- your OS 
- the pattern you use in Validator (a code snippet can help)

Thanks

**Comment by akbacak** [comment]
- There is no folder inside class folders.  They are .../working_directory/videos/classA and B.
  Im using 60 mp4 videos, 30 for each class, from "Diving48 Dataset"

- OS is Ubuntu 18.0, python is 3.6.9
- The Whole code is:
import keras
import os
import glob
from keras_video import VideoFrameGenerator# use sub directories names as classes
classes = [i.split(os.path.sep)[1] for i in glob.glob('videos/*')]
classes.sort()# some global params
SIZE = (112, 112)
CHANNELS = 3
NBFRAME = 5
BS = 8   # Batch Size
# pattern to get videos and classes
glob_pattern='videos/{classname}/'# for data augmentation
data_aug = keras.preprocessing.image.ImageDataGenerator(
    zoom_range=.1,
    horizontal_flip=True,
    rotation_range=8,
    width_shift_range=.2,
    height_shift_range=.2)# Create video frame generator
train = VideoFrameGenerator(
    classes=classes,
    glob_pattern=glob_pattern,
    nb_frames=NBFRAME,
    split=.2,
    shuffle=True,
    batch_size=BS,
    target_shape=SIZE,
    nb_channel=CHANNELS,
    transformation=data_aug,
    use_frame_cache=False)
valid = train.get_validation_generator()

**Comment by metal3d** [comment]
The `glob_pattern` seems to be not right, you should use `videos/{classname}/*.avi` or `videos/{classname}/*` (note the last star char)

**Comment by metal3d** [comment]
For information, you can see how to write glob pattern here: https://docs.python.org/3/library/glob.html

**Comment by akbacak** [comment]
Your rigth, its ok now thx...
